### ISSUE TITLE
NTR: Muscle biopsy: vacuoles (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9063]
New term request Muscle biopsy: vacuoles (segal_040114192540): Vacuolar myopathy; Rimmed vacuoles included  UMLS: 1856349

**Comment by pnrobinson** [comment]
@MickeySegal
is this distinct from [Rimmed vacuoles HP:0003805](https://hpo.jax.org/app/browse/term/HP:0003805) -- it is included according to the above.

**Comment by MickeySegal** [comment]
Muscle vacuoles are not normal.  The question is whether all such vacuoles are rimmed.  I went through many of these diseases for which we list this finding and they tend to list the vacuoles as being rimmed, but some do not: https://www.omim.org/entry/310440 and https://ojrd.biomedcentral.com/articles/10.1186/s13023-017-0646-9 and https://www.omim.org/entry/616231.

So it may be worth having a more general muscle vacuole finding as well as a more specific rimmed vacuole finding.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000977
- label: Muscle vacuoles
- parents: [Abnormal muscle fiber morphology (HP:0004303)](https://hpo.jax.org/app/browse/term/HP:0004303)
- pmid: 
- def: Presence of abnormal vacuoles (membrane-bound organelles) in the sarcolemma. The vacuoles are often associated with cytoplasmic and occasionally intranuclear eosinophilic inclusions.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9063
- synonyms: 

Adding new term: 
- ID: HP:6000978
- label: Nonrimmed muscule vacuoles
- parents: [Abnormal muscle fiber morphology (HP:0004303)](https://hpo.jax.org/app/browse/term/HP:0004303)
- pmid: 
- def: Presence of abnormal vacuoles (membrane-bound organelles) in the sarcolemma. On histological staining with hematoxylin and eosin, non-rimmed vacuoles lack the prominent dark border (rim) seen in rimmed vacuoles.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9063
- synonyms:
